cli/provider/cmd.go

```diff
@@ -155,6 +155,10 @@ func (c *CmdConfigurator) AddFlags(cmd *cobra.Command, cfg *config.Config) error
 	switch cmd.Name() {
 	case "update":
 		return nil
+	case "normalise":
```

Comment:
I think test report name (test-set-0-report) and test case name is enough to get to this normalisation because you can get whole information from that. And please add these required things in config also.

Reply:
Yes, those two flags would be compulsory, i.e. the test-report-name and the test cases(prural) to be normalised. On top of them, i thought adding the --path flag would allow the user to provide the path to the test report folder incase it wasn't the default "./".

Reply:
Also could you elaborate on what you mean by adding these to config?

Reply:
Actually we should give option to the user to provide the test-run also so the flags would test-run, test-set, test-case. regarding path flag it should be normal path flag which we get in record and replay commands. It shouldn't be specific to reports. We have a config file which will be used for provided info like path etc without flags. There is a package for it named config.

Reply:
Refactored